src/core/core.datasetController.js

```diff
@@ -268,6 +268,39 @@ helpers.extend(DatasetController.prototype, {
 		}
 	},
 
+	/**
+	 * Returns a set of predefined style properties that should be used to represent the dataset
+	 * or the data if the index is specified
+	 * @params {number} index - data index
+	 * @return {IStyleInterface} style object
+	 */
+	getStyle: function(index) {
```

Comment:
@nagix I know it comes from [my suggestion](/chartjs/Chart.js/pull/5621#issuecomment-463627999) but do you think it really needs to be public? I'm fine with it, just want your thought (@etimberg @kurkle @benmccann?)

Reply:
I probably would have gone with making it private since it's new just to give any issues a chance to shake out, but don't feel strongly

Reply:
I think its fine being public.